scripts/make_port_status.py

```diff
@@ -120,7 +121,7 @@ def get_mathlib4_module_commit_info(contents):
 sync_prs = defaultdict(set)
 mathlib4repo = github.Github(github_token).get_repo("leanprover-community/mathlib4")
 for pr in mathlib4repo.get_pulls(state='open'):
-    if pr.created_at < datetime.datetime(2022, 12, 1, 0, 0, 0):
+    if pr.created_at < datetime.datetime(2022, 12, 1, 0, 0, 0, tzinfo=pytz.UTC):
```

Comment:
I'm curious why this was needed.

Reply:
python complained that it can't compare naive with aware datetimes, so I just added this and it worked.

Reply:
This is the cause I suppose:
https://github.com/PyGithub/PyGithub/pull/2565